RobotsTxtParser.php

```diff
@@ -157,7 +157,7 @@ private function prepareRules()
 			$directive = trim(strtolower($parts[0]));
 			$value = trim($parts[1]);
 
-			if (!in_array($directive, $this->getAllowedDirectives()) || !$value) {
+			if (!in_array($directive, $this->getAllowedDirectives(), true)) {
```

Comment:
Do you think we really need a strict comparison? I understand that it also works, so ok :)

Reply:
It's faster 😁